CMakeLists.txt

```diff
@@ -854,7 +854,7 @@ ENDIF()
 # ---------------------------------------------------------------------------- #
 
 #
-# Helper function that checks that we can compile and link _src with the given
+# Helper function which checks that we can compile and link _src with the given
```

Comment:
```suggestion
# Helper function that checks that we can compile and link _src with the given
```
"that" is correct usage here. 😁

Reply:
I shouldn't leave UNC until I get this right!